pkg/certsuite/certsuite.go

```diff
@@ -21,14 +22,19 @@ import (
 	"github.com/test-network-function/cnf-certification-test/pkg/provider"
 )
 
-func LoadChecksDB() {
+func LoadChecksDB(labelsExpr string) {
 	accesscontrol.LoadChecks()
 	certification.LoadChecks()
 	lifecycle.LoadChecks()
 	manageability.LoadChecks()
 	networking.LoadChecks()
 	observability.LoadChecks()
 	performance.LoadChecks()
+
+	if preflight.ShouldRun(labelsExpr) {
+		logrus.Infof("Loading preflight's lib checks.")
```

Comment:
This log looks redundant with the one already present inside `preflight.LoadChecks()`. Also, I'm changing those logs from "Entering [suite] suite" to "Loading [suite] checks", which I think describes better the task.